protocols/readme.md

```diff
@@ -0,0 +1,28 @@
+## Off-chain protocols
+
+This package defines the off-chain protocols for managing channels. These protocols are abstract rules which clients should follow in order to reach a desired goal.
+
+To implement the protocols, we make use of an abstraction we call the `Objective`. Objectives are named for the goal which the off-chain protocol aspires to achieve.
+
+The following table shows some example objectives and their implemetation status:
+
+| Objective type      | Implemented |
+| ------------------- | ----------- |
+| `direct-funding`    | x           |
+| `direct-defunding`  |             |
+| `virtual-funding`   |             |
+| `virtual-defunding` |             |
```

Comment:
nit: It almost sounds like `directly-fund` is a better name for `Objective`. hard to imagine this is worth the refactor though?

Reply:
I agree that's a better name. Let's think about if and when to make the change?

Reply:
yep, this is not a blocker at  all